shared/utils/relabel-ids.py

```diff
@@ -483,13 +497,14 @@ def main():
             checkcontentref.set("href", newocilfile)
             checkexport = check.find("./{%s}check-export" % xccdf_ns)
             if checkexport is not None:
-                newexportname = translator.generate_id("{" + oval_ns + "}variable",
+                newexportname = translator.generate_id("{" + ocil_ns + "}variable",
                                                        checkexport.get("export-name"))
```

Comment:
This code was broken, maybe it was not used at all? BTW does `check.get("system") != oval_cs` imply `check.get("system") == oval_cs`